### PR TITLE
Feature/home

### PR DESCRIPTION
**Pull request by Alfianxx**
create home UI

### CODE REVIEW

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            android:background="?attr/colorPrimary"
+            app:popupTheme="@style/ThemeOverlay.MaterialComponents.Light" />
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <com.synnapps.carouselview.CarouselView
+        android:id="@+id/carousel_view"
+        android:layout_width="match_parent"
+        android:layout_height="200dp"
+        app:fillColor="#FFFFFFFF"
```
Comment (akbard838) [inline app/src/main/res/layout/fragment_home.xml]: change this to color

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        app:layout_constraintTop_toBottomOf="@id/appBarLayout"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:pageColor="#00000000"
```
Comment (akbard838) [inline app/src/main/res/layout/fragment_home.xml]: change this to color

File: app/src/main/res/layout/article_item.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_marginStart="8dp"
+    android:layout_marginTop="4dp"
```
Comment (akbard838) [inline app/src/main/res/layout/article_item.xml:8]: add dimen

File: app/src/main/res/layout/article_item.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_marginStart="8dp"
```
Comment (akbard838) [inline app/src/main/res/layout/article_item.xml:7]: add dimen

File: app/src/main/res/layout/article_item.xml (hunk 3 above)
Comment (Alfianxx) [reply app/src/main/res/layout/article_item.xml:8]: udah di fix

**Pull request merged**